docs/geozarr-eopf-specification.qmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "GeoZarr for EOPF - Experimental Specification"
+subtitle: "Transforming Earth Observation Processing Framework Zarr to GeoZarr V3"
+author: "GeoZarr Examples Project"
+date: today
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    code-fold: false
+    code-tools: true
+---
+
+## Overview
+
+This document outlines the experimental GeoZarr specification for transforming EOPF (Earth Observation Processing Framework) Zarr stores into GeoZarr V3 compliant datasets. The specification is based on the implementation in `06_multiscales_as_WebMercatorQuad_EOPFZarrV3.ipynb` and demonstrates how GeoZarr could be adapted for Zarr format 3 while maintaining compatibility with existing geospatial tools.
+
+GeoZarr V3 extends the Zarr V3 format to support geospatial data with proper coordinate reference systems, multiscale overviews, and CF-compliant metadata, specifically addressing the gaps found in EOPF Zarr format.
+
+## Gaps in EOPF Zarr Format
+
+The current EOPF Zarr stores have several gaps that prevent them from being fully GeoZarr compliant:
+
+### Issues Identified
+
+| Issue | Description | Impact |
+|-------|-------------|---------|
+| **Missing CF Standard Names** | Variables lack `standard_name` attributes | Reduces interoperability with CF-compliant tools |
```

Comment:
What proportion of the EOPF data be accurately described by the existing [CF standard names](https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html)? I was under the impression that data variable standardization was a bit more challenging with satellite products relative to climate / forecasting data.